wgpu-core/src/device/mod.rs

```diff
@@ -1939,6 +1939,19 @@ impl<B: GfxBackend> Device<B> {
                         },
                     );
                 }
+
+                if let wgt::VertexFormat::Double
```

Comment:
Given the feedback from @cwfitzgerald , we can't have these formats exposed behind the feature. We'll either need to remove them, or to have a separate issue for them specifically (something like `VERTEX_ATTRIBUTE_64BIT`), that potentially aggregates all of u64, i64, and f64 that we'll expose.
How do you feel about this? Sorry for turning back here, my initial suggestion was wrong.

Reply:
Either way seems fine to me

Reply:
Then splitting into two features seems like the way to go, that way we can have both things possible in wgpu and won't have to revisit this later.

Reply:
How would we detect whether the backend supports 64-bit vertex attributes?

Reply:
We'd need to call [format_properties](https://github.com/gfx-rs/gfx/blob/00285817af676e091ac886b927e9e5e292f22a03/src/hal/src/adapter.rs#L103) on each of the vertex formats exposed by this potential feature, and ensure that all of them have [BufferFeature::VERTEX](https://github.com/gfx-rs/gfx/blob/00285817af676e091ac886b927e9e5e292f22a03/src/hal/src/format.rs#L194). Saying that, this will be some new code, and it's totally fine if you just make it report as unsupported in this PR, instead. Up to you :)

Reply:
I think I'll just add the feature but leave it unsupported then